assets/sass/bootstrap_import/_bootstrap_override.scss

```diff
@@ -127,6 +127,37 @@
             transform: translate(0, -50%);
         }
     }
+    &.btn-icon-chevron-left {
```

Comment:
I understand what we have here but I would suggest separating concerns regarding buttons with icons in general and this particular example:

&[class*="btn-icon-"] {
    // general rules for any button that has icon in it, like gap, &after rules, aligning etc
}

&.btn-icon-chevron-left {
    &:after {
        @extend %icon-chevron-left; // only the particular icon rendering
    }
}


that way we can easily add new icons in buttons and coloring/styles are not involved only with class btn-icon-chevron-left